### DISCUSSION TITLE
Se fala de acessibilidade, mas a documentação só contempla a língua inglesa.

### DISCUSSION THREAD

**Discussion by nazarepiedady** [frontendbr/forum#2023] [Discussão]
## Leia apesar do título não ser muito descritivo

Todo mundo que hoje tem mais de 5 anos ou menos de experiência e que já domina o inglês sabe ou ao menos deveria saber o quão dificil é para um iniciante estudar e aprender uma determinada tecnologia cuja documentação apenas se encontra em inglês, e de qual miserável é as iniciativas de tornar possível o prendizado por parte daqueles que não se localizam em **zonas de fácil**, com zonas de fácil acesso estou me referindo a capitais onde é fácil achar boas escolas de língua, ou mesmo não tenha um orçamento que possa cobrir cursos online, e é quase impossível se sustentar só conteúdo gratuito, muito do aprendizado é processo personalizado. É claro todo desenvolvedor que se preze terá que dominar a certo nível o idioma move a sua área, contudo não adianta muito falar em **comunidades fortes** se a documentação de tecnologias não se encontra disponíveis na linguagem desses supostas **"comunidades fortes"**.

Muitas tecnologias que tiveram seu processo de tradução da documentação iniciadas mas paralizaram, não sei o porque, e muitas que aparentemente que se encontram traduzidas estão incompletas ou faltam correções gramaticais e ortográficas.

Eu peço a ajuda de todos os desenvolvedores que utilizam determinada tecnologia para tornar essa tecnologia disponível em sua língua materna, é muito fácil citar frases miseráveis de autoajuda tiradas de nossos livros favoritos sobre o assuntos ou copiadas de personalidades que respeitamos ou admiramos, mas considero que é hora de fazer algo que realmente preste e seja duradouro.

**Não importa o quão o inglês usado na documentação seja fácil ou qual pequena seja a curva de aprendizado da tecnologia, estudar a tecnologia na sua língua materna torna ainda mais fácil o aprendizado da tecnologia e ainda mais pequena sua curva de aprendizado**

Só para deixar o exemplo e não falar barato, vou deixar o trabalho que venho fazendo com uma tecnologia que vem crescendo no desenvolvimento back-end feito com a linguagem de programação **Python**, [Flask](https://github.com/nazarepiedady/flask-dc).

**Comment by willycamargo** [comment]
Sofri muito para consumir conteúdo no começo de meus estudos, uns 12 anos atrás. Faltava conteúdo em português e eu mal sabia como pesquisar em inglês, quem dera me aprofundar em conceitos técnicos.

Mas hoje tenho uma visão de que apenas conteúdos básicos realmente precisam estar na lingua materna. Cursos introdutórios, documentações de linguagens (e não frameworks), tutoriais básicos etc.

A tecnologia evolui muito rápido, a documentação é, muitas vezes, praticamente viva. E os desenvolvedores preocupados em colaborar com open source normalmente estão muito mais interessados em corrigir bugs ou implementar features do que traduzir documentações. Afinal, a maioria de nós somos apenas programadores e não tradutores.

Minha dica para quem precisa ou quer estudar uma tecnologia que não possua uma tradução para seu idioma materno: tente utilizar o translate para traduzir frase por frase. Aproveite esse momento para aprender um pouco de inglês enquanto aprende a tecnologia.

Traduzir aos poucos enquanto estuda vai fazer tudo ficar mais difícil e cansativo, provavelmente 2 ou 3 vezes mais difícil. Mas a longo prazo isso pode ser bastante útil e você não irá se arrepender.

Finalizando, eu acho admirável quem traduz documentações como você @nazarepiedady e concordo que poderíamos melhorar isso, mas também entendo o motivo dessas traduções não serem feitas por outras pessoas e por não existirem na maioria dos frameworks ou bibliotecas.

**Reply by nazarepiedady** [reply]
@willycamargo o que você está querendo me dizer é praticamente que é cada um por sí no mundo da tecnologia, se eu sofri, os outros devem também sofrer para chegar em alturas que cheguei?

Nós que trabalhamos com tecnologia da informação, temos muito tempo livre porque somos capazes de automatizar varias tarefas em quase todos os aspectos de nossas atividades. Custa ajudar a traduzir uma documentação? funcionalidades são coisas pequenas adicionadas a tecnologia que já existe com muitas outras outras tecnologias, mas com a documentação traduzida. a única preocupação será cobrir a pequenos pedaços de documentação que dizem respeitas as funcionalidades que foram acrescentadas.

> Minha dica para quem precisa ou quer estudar uma tecnologia que não possua uma tradução para seu idioma materno: tente utilizar o translate para traduzir frase por frase. Aproveite esse momento para aprender um pouco de inglês enquanto aprende a tecnologia.

A tradução não é só para mim, eu entendo inglês e falo inglês.

> Traduzir aos poucos enquanto estuda vai fazer tudo ficar mais difícil e cansativo, provavelmente 2 ou 3 vezes mais difícil. Mas a longo prazo isso pode ser bastante útil e você não irá se arrepender.

Não seria melhor tornar o processo menos cansativo?
Porquê que com a possibilidade fazer o melhor, as pessoas tem que escolher o pior?

Era só dizer que não tem nenhum interesse em ajudar, não arranje desculpas como essas, para dar uma legal enquanto não faz nada para ajudar.

**Reply by willycamargo** [reply]
Não, não é cada um por sí e mesmo relendo não acho que me expressei tão mal para que você interpretasse isso.

Não quero ficar traduzindo documentações porque essa não é uma atividade que me dá prazer e não acho que tenho que me obrigar a fazer o que não gosto.

Admiro seu trabalho e o parabenizo por isso. Mas no meu tempo livre eu prefiro ensinar programação diretamente para crianças carentes com instruções fáceis e didáticas. 

Desde o começo da minha carreira eu estive em contato com a educação e, apesar de não depender disso financeiramente, ensino, divulgo e viabilizo a educação de programação e inglês para um grupo de crianças e adolescentes. Entende-se viabilizar como tirar do meu próprio bolso tudo que é necessário para que ela possa estudar: desde computadores, internet e materiais.

Minha visão de acessibilidade e oportunidades é diferente da sua. Acho mais produtivo a longo prazo, como eu disse anteriormente, educar para que a pessoa não tenha dependência de materiais em português do que tentar mudar a cultura de todo um setor que nasceu como a linguagem mãe baseado no inglês.

Volto a ressaltar que acho IMPORTANTÍSSIMO conteúdos que abordam conceitos básicos e iniciais na língua materna. Mas a partir de um ponto um bom profissional vai precisar beber direto da fonte sem precisar de um profissional como proxy tradutor.

Se quiser continuar explorando o assunto de forma saudável e sem distorcer ou tirar conclusões precipitadas sobre as ideias, estou aqui. Inclusive estou a disposição para ler artigos e pesquisas sobre quais ações causam mais impacto em acessibilidade e diversidade dentro de nossa área.

**Reply by wilsonneto-swe** [reply]
Caraca @nazarepiedady, o cara escreveu no melhor tom possível e uma opinião totalmente amigável e concordando com você, e o que você entendeu foi "é cada um por si no mundo da tecnologia"... Nem na mesma língua a gente se entende né?!... Concordo totalmente com o @willycamargo e discordo totalmente da sua postura na resposta a ele...

```
Era só dizer que não tem nenhum interesse em ajudar, não arranje desculpas como essas, para dar uma legal enquanto não faz nada para ajudar.
```

Estamos nos ajudando aqui neste fórum, não estamos? É apenas com traduções que se ajuda os outros?

Estamos no mesmo barco, o @willycamargo , eu e você concordamos que tradução é importante. Por que tentar forçar que a linha de pensamento dele foi contrária e extrema como sua reposta? Apenas para reforçar seu ponto de vista? Ah mano, aceitar a opinião dos amiguinhos faz parte, e tudo é open source, porque assim em vez de cobrar dos outros com grosseria você mesmo pode ir lá e traduzir...

Somos colegas de profissão, aceitar opiniões contrárias e opiniões que concordam com a sua faz parte... Interpretação de fala ou texto, espirito de equipe e comunidade, ver outros pontos de vista, comunicação amigável e não violenta, postura positiva e construtiva... isso sim ajuda muito mais a comunidade do que traduzir doc's, e é muito mais importante do que skills técnicas.

Apenas minha opinião, e sobra as traduções, sim são necessárias, concordo com **vocês dois**...
